lib/grant-types/abstract-grant-type.js

```diff
@@ -46,9 +50,12 @@ AbstractGrantType.prototype.generateAccessToken = function() {
  * Generate refresh token.
  */
 
-AbstractGrantType.prototype.generateRefreshToken = function() {
+AbstractGrantType.prototype.generateRefreshToken = function(client, user, scope) {
   if (this.model.generateRefreshToken) {
-    return Promise.try(this.model.generateRefreshToken);
+    return Promise.try(promisify(this.model.generateRefreshToken, 3), [client, user, scope])
+      .then(function(refreshToken) {
+        return refreshToken || tokenUtil.generateRandomToken();
```

Comment:
Do you think it's a good idea to fall back to the default `generateRandomToken`?
I would consider it an error if `model#generateRefreshToken` is implemented but returns a falsy value.